src/components/itemsList.js

```diff
@@ -1,20 +1,46 @@
 /* eslint-disable react/button-has-type */
 /* eslint-disable react/prop-types */
 /* eslint-disable react/destructuring-assignment */
-// const apiURl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/IwD6i2qVfkciDWYsseU9/books';
+import { buildStyles, CircularProgressbar } from 'react-circular-progressbar';
 
 function ItemsList(props) {
+  const percentage = Math.floor(Math.random() * 100 + 1);
   return (
     <ul className="list">
```

Comment:
- [ ] I kindly suggest that you ensure your follow the Zeplin design given by:

- Ensure your list spans the whole page and is aligned vertically to the bookstore CMS and Profile Icon
- Ensure that you use the same font or similar font as given in the Zeplin design
- Ensure that your button has the same font style and font weight as seen in the Zeplin design
- Ensure that there is an horizontal bar after your comments and remove/delete button as seen in the Zeplin design.



| First Header  | Second Header |
| ------------- | ------------- |
| <img width="911" alt="Screenshot 2022-09-03 145915" src="https://user-images.githubusercontent.com/52934128/188273848-3de7dbcc-f696-4562-bd58-e06daa19085b.png"> | <img width="902" alt="image" src="https://user-images.githubusercontent.com/52934128/188273780-19c7ce40-04f7-433e-bea4-76bb2616cd29.png"> | 
| Content Cell  | Content Cell  |